examples/next/package.json

```diff
@@ -21,6 +19,8 @@
     "react-map-gl": "7.0.15"
   },
   "devDependencies": {
+    "@types/node": "^15.12.4",
+    "@types/react": "^17",
```

Comment:
I moved these to dev dependencies because they should not be in the dependencies.